auth0/src/test/java/com/auth0/android/provider/WebAuthProviderTest.java

```diff
@@ -2437,6 +2451,20 @@ public void shouldHaveReturnToUriOnLogout() {
         assertThat(uri.getQueryParameter("returnTo"), is("https://test.domain.com/android/com.auth0.android.auth0.test/callback"));
     }
 
+    @Test
+    public void shouldSetRedirectToUrlIgnoringSchemeOnLogout() {
```

Comment:
Shouldn't be `shouldSetReturnToUrlIgnoringSchemeOnLogout `?

Reply:
Good catch! 🥇